CODEOWNERS

```diff
@@ -291,6 +291,8 @@ extensions/filters/http/oauth2 @derekargueta @snowp
 /*/extensions/load_balancing_policies/round_robin @wbpcode @UNOWNED
 # Early header mutation
 /*/extensions/http/early_header_mutation/header_mutation @wbpcode @UNOWNED
+# IP Geolocation
+/*/extensions/filters/http/geoip @nezdolik @mattklein123
```

Comment:
@mattklein123 i will update the ownership once it's clear who is sponsoring the change

Reply:
@ravenblackx should i put you instead of Matt as an owner?

Reply:
I guess. It's not really my field, but it would make sense since having reviewed some of it I now have the best context to review the rest of it.